code/modules/clothing/glasses/_glasses.dm

```diff
@@ -70,8 +70,10 @@
 	if(ishuman(user) && slot == ITEM_SLOT_EYES)
 		ADD_TRAIT(user, TRAIT_MADNESS_IMMUNE, CLOTHING_TRAIT)
 
-/obj/item/clothing/glasses/meson/dropped(mob/user)
-	. = ..()
+/obj/item/clothing/glasses/meson/dropped(mob/living/carbon/human/user)
```

Comment:
the parent function argument type is `mob/`, so ideally good practice would be to do the cast inline instead, just because people tend to copy-paste proc overrides.

```
/obj/item/clothing/glasses/meson/dropped(mob/user)
    ..()
    if(ishuman(user))
       var/mob/living/carbon/human/H = user
       if(H.glasses != src)
             return
    else
       return
```
even if it looks uglier